### PR TITLE
Rob dev comment styling

### PR DESCRIPTION
**Pull request by rob-marcus**
Comments with styleeeeee. 
Pics: 
Screen Shot 2020-06-16 at 1 12 11 PM

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/UpvoteCommentServlet.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    Key taskEntityKey = KeyFactory.createKey("Comment", id);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.delete(taskEntityKey);
+
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("comment", comment);
+    commentEntity.setProperty("timestamp", timestamp);
+    commentEntity.setProperty("upvotes", upvotes);
+    commentEntity.setProperty("id", id);
+
```
Comment (ailemeo) [inline portfolio/src/main/java/com/google/sps/servlets/UpvoteCommentServlet.java:60]: This looks very different from the upVote logic we worked on in the previous PR. Confirm that you have merged correctly with master.

Comment (rob-marcus) [reply portfolio/src/main/java/com/google/sps/servlets/UpvoteCommentServlet.java:60]: Oops! Was waiting to merge upvote feature and then do the resolve merge conflict on this file; forgot to do that yesterday afternoon. Yes, merged correctly, this just needs to be caught up. Doing now.

Comment (rob-marcus) [reply portfolio/src/main/java/com/google/sps/servlets/UpvoteCommentServlet.java]: Finished merging and such; I also updated the upvote logic per your last comment. It should now handle simultaneous upvoting.

File: portfolio/src/main/java/com/google/sps/servlets/UpvoteCommentServlet.java [hunk 2]
```diff
@@ -43,13 +43,13 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     long id = Long.parseLong(request.getParameter("id"));
-    long upvotes = Long.parseLong(request.getParameter("upvoteCount"));
-
+    long oneUpvote = Long.valueOf(1);
```
Comment (andrewlathamgoogle) [inline portfolio/src/main/java/com/google/sps/servlets/UpvoteCommentServlet.java:46]: what is the point of this?  Just do upvotes += 1

File: portfolio/src/main/webapp/comments.js [hunk 3]
```diff
@@ -69,59 +68,78 @@ function addMessage(pageNumber = 0, sortMethod = {"feature": "timestamp",
   });
 }
 
+
 /**
  * Build the elements of an individual message using JSON data
  * Also create a delete button
  */
 function createMessageElements(comment) {
```
Comment (andrewlathamgoogle) [inline portfolio/src/main/webapp/comments.js:76]: this method is really long and becomes confusing to read with so many similarly-named elements.  break it up into smaller methods for each element.

**Pull request merged**